lib/schematic/CircuitJsonToKicadSchConverter.ts

```diff
@@ -26,8 +26,36 @@ export class CircuitJsonToKicadSchConverter {
     const KICAD_CENTER_X = 95.25
     const KICAD_CENTER_Y = 73.66
 
+    const db = cju(circuitJson)
+
+    // Calculate the center of the schematic components to center the schematic
+    // on the page.
+    const schematicComponents = db.schematic_component.list()
+
+    let minX = Infinity
+    let minY = Infinity
+    let maxX = -Infinity
+    let maxY = -Infinity
+
+    if (schematicComponents.length > 0) {
+      for (const component of schematicComponents) {
+        minX = Math.min(minX, component.center.x)
+        minY = Math.min(minY, component.center.y)
+        maxX = Math.max(maxX, component.center.x)
+        maxY = Math.max(maxY, component.center.y)
+      }
+    } else {
+      minX = 0
+      minY = 0
+      maxX = 0
+      maxY = 0
+    }
```

Comment:
this bounds calculation isn't very accurate, we should introduce a function to `@tscircuit/circuit-json-util`, but if you're going to keep it in this module, please put it in a separate file! `getSchematicBounds(circuitJson)`

Reply:
to make it more accurate, you would need to add/substract width/2 and height/2 to your min/max calculations. You also need to consider all points along each schematic_trace path